apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -24,16 +24,20 @@ GameLab.prototype.init = function(config) {
 };
 
 GameLab.prototype.resetHandler = function(ignore) {
-  $('.droplet-main-canvas').css('background-color', '#FFF');
-  $('.droplet-transition-container').css('background-color', '#FFF');
-  $('.ace_scroller').css('background-color', '#FFF');
+  if (!this.studioApp_.config.readonlyWorkspace) {
+    $('.droplet-main-canvas').css('background-color', '#FFF');
+    $('.droplet-transition-container').css('background-color', '#FFF');
+    $('.ace_scroller').css('background-color', '#FFF');
+  }
   P5Lab.prototype.resetHandler.call(this, ignore);
 };
 
 GameLab.prototype.runButtonClick = function() {
-  $('.droplet-main-canvas').css('background-color', '#E5E5E5');
-  $('.droplet-transition-container').css('background-color', '#E5E5E5');
-  $('.ace_scroller').css('background-color', '#E5E5E5');
+  if (!this.studioApp_.config.readonlyWorkspace) {
+    $('.droplet-main-canvas').css('background-color', '#E5E5E5');
+    $('.droplet-transition-container').css('background-color', '#E5E5E5');
+    $('.ace_scroller').css('background-color', '#E5E5E5');
```

Comment:
Since these colors are going to be used in lots of places, consider adding them to our `shared/css/color.scss` file and referencing it from `@cdo/apps/util/color`. (Looks like white is already there)